superset/views/database/views.py

```diff
@@ -130,24 +130,29 @@ def form_get(self, form: CsvToDatabaseForm) -> None:
 
     def form_post(self, form: CsvToDatabaseForm) -> Response:
         database = form.con.data
-        csv_table = Table(table=form.name.data, schema=form.schema.data)
+
+        if "." in form.name.data:
```

Comment:
This is a detail, but I think it is worth moving the format checking to a separate method / function, so that the name of the method says what you really want to check and hide the implementations here to improve readability. It currently says you want to check if it contains a period, but in fact you are checking to see if it contains a namespace.

Reply:
@ad-m I've restructured the PR using an alternative formulation.